vendor/runner/cli.js

```diff
@@ -0,0 +1,101 @@
+// Usage:
+//
+// Run all vendors:
+//   npm run test:vendor
+//
+// Run some vendors:
+//   npm run test:vendor -- --only=vendor1,vendor2,vendor3
+//
+// Infer the current PR from the Travis environment, and look for bracketed,
+// space-separated vendor names in the pull request title. If none are found,
+// do not run any tests. For example:
+// Pull request title: [travis sonar] Support user token authentication
+//   npm run test:vendor -- --pr
+// is equivalent to
+//   npm run test:vendor -- --only=travis,sonar
+
+'use strict';
+
+const request = require('request');
```

Comment:
Done, though I prefer to automate this kind of thing. It's too fussy. ESLint has a rule we could turn on eventually.